Common/Securities/SymbolPropertiesDatabase.cs

```diff
@@ -108,17 +109,32 @@ public bool TryGetMarket(string symbol, SecurityType securityType, out string ma
         /// Gets the symbol properties for the specified market/symbol/security-type
         /// </summary>
         /// <param name="market">The market the exchange resides in, i.e, 'usa', 'fxcm', ect...</param>
-        /// <param name="symbol">The particular symbol being traded</param>
+        /// <param name="symbol">The particular symbol being traded (Symbol class)</param>
         /// <param name="securityType">The security type of the symbol</param>
         /// <param name="defaultQuoteCurrency">Specifies the quote currency to be used when returning a default instance of an entry is not found in the database</param>
         /// <returns>The symbol properties matching the specified market/symbol/security-type or null if not found</returns>
-        public SymbolProperties GetSymbolProperties(string market, string symbol, SecurityType securityType, string defaultQuoteCurrency)
+        /// <remarks>For any derivative options asset that is not for equities, we default to the underlying symbol's properties if no entry is found in the database</remarks>
+        public SymbolProperties GetSymbolProperties(string market, Symbol symbol, SecurityType securityType, string defaultQuoteCurrency)
         {
             SymbolProperties symbolProperties;
-            var key = new SecurityDatabaseKey(market, symbol, securityType);
+            var lookupTicker = symbol?.SecurityType == SecurityType.Equity ? symbol.Value : symbol?.ID.Symbol;
```

Comment:
not sure about using `symbol.Value`, this might change -> be remapped. Maybe we should use similar logic as before calling `MarketHoursDatabase.GetDatabaseSymbolKey(symbol)` ?

Reply:
This resolves to the same logic we had before, but with additional support for FOPs and other derivatives we might have in the future.